db/migrate/20210131233252_add_bitflyer_deposit_id_to_publishers.rb

```diff
@@ -0,0 +1,5 @@
+class AddBitflyerDepositIdToPublishers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :publishers, :bitflyer_deposit_id, :string
```

Comment:
Instead of creating a new column, can we use a table like UpholdConnections that stores deposit id and the publisher id? I don't like adding more columns to already bloated tables.

Reply:
You're totally right @yachtcaptain23 , Fixed, moved this field out of the publishers table